Cargo.toml

```diff
@@ -26,7 +26,7 @@ flate2 = "1.0.28"
 serde = ["dep:serde", "dep:serde_with", "ordered-float/serde", "indexmap/serde"]
 
 [dev-dependencies]
-serde_json = { version = "1.0.96", features = ["float_roundtrip"] }
+serde_json = { version = "1.0.96", features = ["float_roundtrip", "preserve_order"] }
```

Comment:
The tests pass even without this feature, so I'm not 100% sure if I'm using this properly.